bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v19/BitcoindV19RpcClientTest.scala

```diff
@@ -0,0 +1,62 @@
+package org.bitcoins.rpc.v19
+import org.bitcoins.rpc.client.common.BitcoindVersion
+import org.bitcoins.rpc.client.v19.BitcoindV19RpcClient
+import org.bitcoins.testkit.rpc.BitcoindRpcTestUtil
+import org.bitcoins.testkit.util.BitcoindRpcTest
+
+import scala.concurrent.Future
+
+class BitcoindV19RpcClientTest extends BitcoindRpcTest {
+  lazy val clientF: Future[BitcoindV19RpcClient] = {
+    val client = new BitcoindV19RpcClient(BitcoindRpcTestUtil.v19Instance())
+    val clientIsStartedF = BitcoindRpcTestUtil.startServers(Vector(client))
+    clientIsStartedF.map(_ => client)
+  }
+  lazy val clientPairF: Future[(BitcoindV19RpcClient, BitcoindV19RpcClient)] =
+    BitcoindRpcTestUtil.createNodePairV19(clientAccum)
+
+  clientF.foreach(c => clientAccum.+=(c))
+
+  behavior of "BitcoindV19RpcClient"
+
+  it should "be able to start a V19 bitcoind instance" in {
+
+    clientF.map { client =>
+      assert(client.version == BitcoindVersion.V19)
+    }
+
+  }
+
+  it should "get a block filter given a block hash" in {
+    for {
+      (client, _) <- clientPairF
+      blocks <- client.getNewAddress.flatMap(client.generateToAddress(1, _))
+      blockFilter <- client.getBlockFilter(blocks.head.hex, "basic")
+    } yield {
+      assert(blockFilter.filter.length > 0)
+      assert(blockFilter.header.length > 0)
```

Comment:
should probably assert that the block filter matches the block and the block filter header matches the block filter. Let me know if you want to know how these are computed.